encodings/sparse/src/canonical.rs

```diff
@@ -199,12 +216,59 @@ fn canonicalize_sparse_decimal<D: NativeDecimalType>(
     Ok(Canonical::Decimal(array))
 }
 
+fn canonicalize_utf8<I: NativePType>(
+    fill_value: Utf8Scalar,
+    indices: Buffer<I>,
+    values: VarBinViewArray,
+    validity: Validity,
+    len: usize,
+) -> VortexResult<Canonical> {
+    let n_patch_buffers = values.buffers().len();
+    let mut buffers = values.buffers().to_vec();
+
+    let fill = if let Some(buffer) = &fill_value.value() {
+        buffers.push(buffer.inner().clone());
+        BinaryView::make_view(
+            buffer.as_ref(),
+            u32::try_from(n_patch_buffers).vortex_expect("too many buffers"),
+            0,
+        )
+    } else {
+        // any <=12 character value will do
+        BinaryView::make_view("FILL_VALUE".as_ref(), 0, 0)
```

Comment:
you can just pass &[], it will make a new Inlined view that is zeroed

Reply:
Done.